look-and-feel.md

```diff
@@ -0,0 +1,65 @@
+# Look and Feel
+
+### Name and wordmark
+
+Our name uses title case with the following variations:
+
+> **HYPHA**  
+> **HYPHA** Worker Co-operative  
+> Hypha Worker Co-operative Inc. [our legal name]  
+
+We use the following wordmark in online spaces, services and social media where we represent the organization (e.g., [Twitter](https://twitter.com/hyphacoop) and [LinkedIn](https://ca.linkedin.com/company/hyphacoop)). 
+
+<img src="images/hypha-wordmark.png" alt="Hypha Wordmark in white on purple background" width="50%"/>
+
+### Typography
+
+Our **heading typeface** is [Work Sans](https://fonts.google.com/specimen/Work+Sans) and our **secondary (body) typeface** is !!!!. For the web we prefer using the [system font stack](https://css-tricks.com/snippets/css/system-font-stack/) sans-serif typeface.
+
+Heading typeface 
+>
+
+Body typeface 
+>
+
+Body typeface (web)
+> `font-family: -apple-system, BlinkMacSystemFont, avenir next, avenir, helvetica neue, helvetica, ubuntu, roboto, noto, segoe ui, arial, sans-serif`
+
+### Colours
+
+<img src="images/hypha-colours.png" alt="Hypha colours" width="90%"/>
+
+Registration Black   
```

Comment:
I associate "Registration Black" to the CMYK process black. I think X11 is just "Black"

Reply:
Okay, was just using the names that I was told were associated!